lib/transaction.go

```diff
@@ -125,6 +126,20 @@ func (o Transaction) Equal(m sebakcommon.Message) bool {
 	return o.H.Hash == m.GetHash()
 }
 
+func (o Transaction) IsValidCheckpoint(checkpoint string) bool {
+	if o.B.Checkpoint == checkpoint {
+		return true
+	}
+
+	a := strings.SplitN(checkpoint, "-", 2)
+	b := strings.SplitN(o.B.Checkpoint, "-", 2)
```

Comment:
Why don't you use [`ParseCheckpoint`](https://github.com/bosnet/sebak/pull/88/files#diff-376d399af0732b484ed220968101362cR163) method instead of `strings.SplitN`?